static/src/javascripts/projects/common/modules/atoms/youtube-player.ts

```diff
@@ -205,6 +205,25 @@ const createAdsConfig = (
 	return adsConfig;
 };
 
+type YTHost = 'https://www.youtube.com' | 'https://www.youtube-nocookie.com';
+const getHost = ({
+	state,
+	containsClass,
+	adFree,
+}: {
+	state: ConsentState;
+	containsClass: (token: string) => boolean;
+	adFree: boolean;
+}): YTHost => {
+	if (
+		canTarget(state) &&
+		!adFree &&
+		containsClass('youtube-media-atom__iframe')
+	)
+		return 'https://www.youtube.com';
```

Comment:
Can we put the return on the same line? And to be extra safe in brackets?

Its quite an important thing to prevent accidentally breaking :)

Reply:
Should probably add https://eslint.org/docs/rules/curly#multi-line

Reply:
I’ve added it inline, at least!